tests/qemu-tests/tests/system/test_commands_kernel.py

```diff
@@ -6,7 +6,8 @@
 
 
 def test_command_kbase():
-    pass  # TODO
+    res = gdb.execute("kbase", to_string=True)
+    assert len(res) > 0
```

Comment:
If we ever decide to print an error or warning message, then this test will always pass. But I don't know a better way to test this, so it's fine for now.

Reply:
We could verify that `found_kbase` matches the address returned by `pwndbg.gdblib.symbol.address("_text")`. However, this approach does not work for our tests involving 5-level paging, apparently because `gdb-pt-dump` lacks support for 5-level paging.

Reply:
> gdb-pt-dump lacks support for 5-level paging

What arch is this for?

Reply:
Support soon to be added - https://github.com/martinradev/gdb-pt-dump/issues/31